.buildkite/_forge.rayci.yml

```diff
@@ -24,6 +24,10 @@ steps:
     wanda: ci/docker/base.ml.wanda.yaml
     depends_on: oss-ci-base_test
 
+  - name: docbuild
+    wanda: ci/docker/doc.build.wanda.yaml
+    depends_on: oss-ci-base_build
```

Comment:
should this be merging master head or something now? feels that this is also doing the docs move.

Reply:
yess, this probably stales now